api/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Core API
+layout: sidebar
+---
+
+## Core API
+
+### Mocha Bindings: [`testium-mocha`](https://www.npmjs.com/package/testium-mocha)
+
+There are two general ways of interacting with the mocha bindings.
+The first (and older) way is to use the `browser` property on the test context:
+
+```js
+var injectBrowser = require('testium-mocha');
+
+before(injectBrowser());
+
+it('has a browser', function() {
+  return this.browser.navigateTo('/');
+});
+```
+
+The problem is that this works very poorly with ES6 arrow functions since it depends on the implicit test context.
+The more ES6-friendly interface is:
+
+```js
+var browser = require('testium-mocha').browser;
+
+before(browser.beforeHook());
+
+it('is the browser', () => browser.navigateTo('/'));
+```
+
+**Warning:** Testium switches out the prototype of `browser` to turn the browser export "magically" into a functioning object after successful initialization.
+Trying to call anything but `beforeHook` before the hook finished will not work as expected.
+
+#### `injectBrowser(options = {})`
+
+Returns a [mocha before hook](http://mochajs.org/#hooks).
+The hook will initialize testium (if it didn't happen already)
+and apply a number of changes to the current test suite:
+
+* Intercept errors and automatically save screenshots.
+* Override the default timeouts using the [`mocha.timeout`](/config.html#mocha-timeout) setting.
+* Install an after hook to ensure everything is properly cleaned up.
+* Inject a `browser` property onto the test context.
+* Switch out the prototype of the [`browser` export](/api/#browser) to point to [`testium.browser`](/api/#testium-browser).
+
+The `options` are forwarder to [`getTestium`](/api/#gettestium-options)
+
+#### `browser`
+
+A proxy to [`testium.browser`](/api/#testium-browser) once the before hook finished.
```

Comment:
Would read more clearly as:

After `beforeHook()` has been executed, this object will be a proxy to [`testium.browser`](/api/#testium-browser).